app/models/profile.rb

```diff
@@ -67,11 +67,11 @@ def activate(reason_deactivated: nil)
   end
   # rubocop:enable Rails/SkipsModelValidations
 
-  def confirm_that_profile_can_be_activated!
+  def reason_not_to_activate
     if pending_reasons.any?
-      raise "Attempting to activate profile with pending reasons: #{pending_reasons.join(',')}"
+      return "Attempting to activate profile with pending reasons: #{pending_reasons.join(',')}"
```

Comment:
I would move the explicit `return` here and in the branch and take advantage of ruby's implicit return
```suggestion
      "Attempting to activate profile with pending reasons: #{pending_reasons.join(',')}"
```